homework_4/part1/SpringIntegrationTestProject/.idea/compiler.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <annotationProcessing>
+      <profile name="Maven default annotation processors profile" enabled="true">
+        <sourceOutputDir name="target/generated-sources/annotations" />
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <module name="SpringIntegrationTestProject" />
+      </profile>
+    </annotationProcessing>
+  </component>
+  <component name="JavacSettings">
+    <option name="ADDITIONAL_OPTIONS_OVERRIDE">
+      <module name="SpringIntegrationTestProject" options="" />
+    </option>
+  </component>
+</project>
```

Comment:
Create .gitignore file and put it in root directory. 
Don't push to git:
- `.idea` folder
- `.iml` files
- `target` folder